.github/workflows/symfony-test.yml

```
@@ -0,0 +1,256 @@
+        with:
+          fetch-depth: 0
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set IMAGE_NAME for this project
+        run: echo "TEST_IMAGE_NAME=${{ needs.build-and-push-test-image.outputs.matrix_test_image_name }}" >> $GITHUB_ENV
```

actionlint (1.7.4)

property "build-and-push-test-image" is not defined in object type {define-matrix: {outputs: {projects: string}; result: string}; install-vendors: {outputs: {}; result: string}}

(expression)